equinox/nn/_spectral_norm.py

```diff
@@ -90,6 +118,13 @@ def __init__(
             The `dtype` of the weight array of the `layer` input is applied to all
             parameters in this layer.
 
+
+        !!! Caution
+
+            If `exact` is true, it computes the transpose via `jax.linear_transpose` of
+            the layer. This includes all operations of the layer call, which means for
+            layers with a bias, this can result in the incorrect spectral value.
```

Comment:
Maybe we should follow JAX's lead here and transpose the tangent pass of `jax.jvp`?

Reply:
Hmmm, I tried to implemented what I thought you meant. This also means we could remove the "weight" flag for the exact case (maybe?) since we basically "determine" the weight through the jvp? 

Let me know if this is what you had in mind, or if I was totally off. Does seem like a lot of jvps tho.